hasjob/views/index.py

```diff
@@ -537,6 +542,29 @@ def browse_tags():
     return render_template('tags.html.jinja2', tags=gettags(alltime=getbool(request.args.get('all'))))
 
 
+@app.route('/f/<name>', subdomain='<subdomain>', methods=['GET', 'POST'])
+@app.route('/f/<name>', methods=['GET', 'POST'])
+def filtered_view(name):
```

Comment:
Add `@FilteredView.is_url_for('view')`

Reply:
I'm not sure how to use this given that we're setting `subdomain` to `None` if `g.board` is `www`.

Reply:
Subdomain is handled automatically.

Reply:
Actually, it won't work for filters. Here's the relevant code: https://github.com/hasgeek/hasjob/blob/master/hasjob/views/board.py#L12-L36